index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+    <title>My_portfolio</title>
+</head>
+<body>
+    <header>
+            <h1>Kandy</h1>
+            <nav>
+                <i class="fa fa-bars"></i>
+            </nav>
+    </header>
+
+    <section class="clm-1">
+        <div class="clm-pres">
+            <h2>I'm Kandy,<br/> Glad to see you!</h2>
+            <h6>I’m a software developer! I can help you build a product , 
+                   feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.</h6>
+        </div>
+        <div class="links">
+            <p>let's connect</p>
+            <ul>
+                <li>
+                    <a href="https://www.linkedin.com/in/kandi-peter-a49590212/">
+                        <img src="./images/Linkedin_icon.png" alt="my linkedin profil">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://github.com/Kandy-Peter">
+                        <img src="./images/Vector_github.png" alt="my github profil">
+                    </a>
+                </li>
+                <li>
```

Comment:
- [ ] Good job on implementing the requirement by using `<ul>` congrats :clap: however I think you forgot one link. Kindly add the `angels` link. check here down.

| your design | the template|
| --------------- | ---------------- |
| ![Screenshot from 2021-11-24 19-20-26](https://user-images.githubusercontent.com/65068771/143285835-6b79edd1-e6a3-4471-94dc-e6e95b8dc0b9.png) | ![Screenshot from 2021-11-24 19-21-35](https://user-images.githubusercontent.com/65068771/143285927-7b503ff5-fc88-4d19-bdc0-24d72b0d8f91.png) |